app/pathogen/arbovirus/dashboard/filters.tsx

```diff
@@ -195,7 +203,7 @@ export function Filters(props: FiltersProps) {
     {field: FilterableField.whoRegion, label: "WHO Region", valueToLabelMap: {}},
     {field: FilterableField.country, label: "Country", valueToLabelMap: {}},
     {field: FilterableField.antibody, label: "Antibody", valueToLabelMap: {}},
-    {field: FilterableField.pathogen, label: "Arbovirus", valueToLabelMap: {}},
+    {field: FilterableField.pathogen, label: "Arbovirus", valueToLabelMap: {}, className: "hidden"},
```

Comment:
I set this to hidden instead of deleting it. It is because this filter is used in the serotype PR in one of those apply filter phases. So we still need it in this list so it can be accessed later (I think).

Reply:
Hmmm not sure if we want to do that. I am sure there is some way around this